libs/ledgerjs/packages/errors/src/index.ts

```diff
@@ -243,7 +243,7 @@ export const StatusCodes = {
   CONDITIONS_OF_USE_NOT_SATISFIED: 0x6985,
   CONTRADICTION_INVALIDATION: 0x9810,
   CONTRADICTION_SECRET_CODE_STATUS: 0x9808,
-  CUSTOM_IMAGE_BOOTLOADER: 0x662f,
+  DEVICE_IN_RECOVERY_MODE: 0x662f,
```

Comment:
can you then rename to `DEVICE_IN_BOOTLOADER_OR_RECOVERY_MODE` ? it's a bit long but since there are 2 use cases 🤷‍♂️